model_compression_toolkit/qat/keras/quantization_facade.py

```diff
@@ -158,7 +158,7 @@ def keras_quantization_aware_training_init(in_model: Model,
 
         user_info.mixed_precision_cfg = bit_widths_config
 
-        return qat_model, user_info, QUANTIZATION_CONFIGS_DICT
+        return qat_model, user_info, None
```

Comment:
Please just add a TODO to remove this None (if I understand correctly, it's not returned anymore since it's replaced by the new load function)